packages/react-reconciler/src/ReactFiberBeginWork.js

```diff
@@ -867,6 +868,46 @@ function deferHiddenOffscreenComponent(
 // fork the function.
 const updateLegacyHiddenComponent = updateOffscreenComponent;
 
+function updateActivityComponent(
```

Comment:
Needs a close review, I have no idea what I'm doing. mountWorkInProgressOffscreenFiber and updateWorkInProgressOffscreenFiber are what Suspense use, so I assume that's how we want to mount and update the children here?

Reply:
Yea this sounds about right.